src/server/components/dashboard/controllers.ts

```diff
@@ -153,12 +155,34 @@ export async function listsController(req: Request, res: Response, next: NextFun
       isNewUser,
       lists: listsWithFormattedDates(lists as ListsForDashboard[]),
       csrfToken: getCSRFToken(req),
+      dashboardBoxes: calculateDashboardBoxes(lists as ListsForDashboard[]),
     });
   } catch (error) {
     next(error);
   }
 }
 
+function calculateDashboardBoxes(lists: ListsForDashboard[]) {
+  return [calculateAdminDashboardBox(lists)];
+}
+
+function calculateAdminDashboardBox(lists: ListsForDashboard[]) {
+  const adminBox = {
+    name: "administrators",
+    text: "All lists have administrators",
+    cssClass: "green",
+  };
+
+  const listsWithNoAdmins = lists.filter((list) => list.admins === 0);
+
+  if (listsWithNoAdmins.length > 0) {
```

Comment:
use pluralize (it's already an installed dependency, so you might as well use it).

pluralize (as well as libs like i18n) will help you use the right "word" when you have 0, 1, or more of something. For example, this will be wrong when there is 1 problematic list. e.g. "1 lists have no administrators".
instead of writing ternaries, you can just do 


```
pluralize("list", count)
// 0 lists have no administrators
// 1 list have no administrators
// 2 lists have no administrators
```

you will also have to add a rule for have/has. (it might already be included. check how `pluralize("has", count)` behaves.) see irregular rules https://www.npmjs.com/package/pluralize